cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -753,6 +775,15 @@ def gen_json_out(self):
                     "poolPrefix": self.out_map['RGW_POOL_PREFIX']
                 }
             })
+        # if 'RGW_TLS_CERT' exists, then only add the "ceph-rgw-tls-cert" secret
+        if self.out_map['RGW_TLS_CERT']:
+            json_out.append({
+                "name": "ceph-rgw-tls-cert",
+                "kind": "Secret",
+                "data": {
+                    "cert": self.out_map['RGW_TLS_CERT'],
```

Comment:
Does rook support the creation of multiple `cephobjectstore` connecting different RGW servers externally in the same `namespace`?  In that case, having the same "ceph-rgw-tls-cert" secret name end up in issues

Reply:
Yes, this should be supported but not necessarily something heavily used. Agreed using the same secret will be an issue. But still I think they are still a few others things missing to make this work.

Looking again at steps 3/4/5 from https://github.com/rook/rook/issues/7804#issuecomment-851915832 we must document these I think. This is not something the operator can do but more the administrator.

So this PR essentially represents steps 5 and 6 will be done by ocs-op.
Do you agree?

Reply:
Step 6 will be done by OCS-OP right?

Reply:
Effectively, step 5 is done by ocs-op yes but that's thanks to the new secret added in the JSON by this PR.